app/src/components/LogViewer.js

```diff
@@ -0,0 +1,65 @@
+// LogViewer script - externalized for CSP compliance
+(function() {
+  'use strict';
+
+  const params = new URLSearchParams(window.location.search);
+  const logPath = decodeURIComponent(params.get('path') || 'Unknown');
+  const logContent = decodeURIComponent(params.get('content') || '');
+
+  // Display log path
+  const logPathElement = document.getElementById('logPath');
```

Comment:
I noticed that this log viewer is reading from C:\Users\<username>\AppData\Roaming\Daydream Scope\logs\main\...

Could you briefly explain why we need to copy server log files from the default location eg ~/.daydream-scope/logs into this directory? Not against it just want to understand what is going on.

Reply:
Hmm seems simpler to have a single source of truth re: logs and then have the LogViewer open the latest log file from wherever the server keeps logs. This assumes that if you install from source and use the Electron app that by default they share the same ~/.daydream-scope dir which feels fine because you can always configure this to be elsewhere too.

So, I'd say just to simplify to read from ~/.daydream-scope. WDYT?